notebooks/book2/29/STS_TFP.ipynb

```diff
@@ -0,0 +1,1573 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        demand_dates[anomalies], demand[anomalies], c="red", marker="x", s=20, linewidth=1, label=r"Anomalies (&gt;3$\sigma$)"
</pre><p>It showing reversed question mark instead of "&gt;"</p><p>Try this - "Anomalies$&gt;3\sigma$" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>